fastchat/utils.py

```diff
@@ -5,9 +5,12 @@
 import json
 import logging
 import logging.handlers
+import numpy as np
```

Comment:
Organize the import statements.

1. builtin libraries
2. 3rd party libraries
3. fastchat libraries

torch and np should go to item 2.